benefits/core/templates/core/index.html

```diff
@@ -11,6 +11,6 @@
 
 {% block call-to-action %}
   {% translate "core.pages.index.button" as trigger_text %}
-  {% include "core/includes/modal-trigger.html" with id="agency-selector" classes="btn btn-lg btn-primary" text=trigger_text %}
-  {% include "core/includes/modal--agency-selector.html" with id="agency-selector" %}
+  {% include "core/includes/modal-trigger.html" with id="modal--agency-selector" classes="btn btn-lg btn-primary" text=trigger_text %}
+  {% include "core/includes/modal--agency-selector.html" with id="modal--agency-selector" %}
```

Comment:
Confirmed this fix caused this issue: #1618 

The new ID needs to be used in style.css for these lines:
<img width="485" alt="image" src="https://github.com/cal-itp/benefits/assets/3673236/a1156537-9c24-407a-93d1-d1f8da5fa417">